backend/apps/slack/tests/test_slack_sync_data.py

```
@@ -0,0 +1,57 @@
+from django.test import TestCase
```

Ruff (0.11.9)

File `backend/apps/slack/tests/test_slack_sync_data.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)